packages/api-tests/test/browser-test-setup.js

```diff
@@ -0,0 +1,70 @@
+const webdriverio = require('webdriverio');
+const selenium = require('selenium-standalone');
+
+const options = {
+  desiredCapabilities: {
+    browserName: 'chrome',
+    chromeOptions: {
+      // Disables "Chrome is being controlled by testing software" banner.
+      args: ['disable-infobars']
+    }
+  }
+};
+
+// This is designed to replicate the Spectron Application object, apart from launching chrome instead of Electron.
```

Comment:
So we don't need to change the tests. We use `app.start()`, `app.stop()` etc in the beforeEach and afterEach methods